{
  "Definition": [
    {
      "ID": "oval:org.altlinux.errata:def:20221372",
      "Version": "oval:org.altlinux.errata:def:20221372",
      "Class": "patch",
      "Metadata": {
        "Title": "ALT-PU-2022-1372: package `python3-module-django` update to version 3.2.12-alt1",
        "AffectedList": [
          {
            "Family": "unix",
            "Platforms": [
              "ALT Linux branch p10"
            ],
            "Products": [
              "ALT Server",
              "ALT Virtualization Server",
              "ALT Workstation",
              "ALT Workstation K",
              "ALT Education",
              "Simply Linux",
              "Starterkit"
            ]
          }
        ],
        "References": [
          {
            "RefID": "ALT-PU-2022-1372",
            "RefURL": "https://errata.altlinux.org/ALT-PU-2022-1372",
            "Source": "ALTPU"
          },
          {
            "RefID": "BDU:2022-00584",
            "RefURL": "https://bdu.fstec.ru/vul/2022-00584",
            "Source": "BDU"
          },
          {
            "RefID": "BDU:2022-00585",
            "RefURL": "https://bdu.fstec.ru/vul/2022-00585",
            "Source": "BDU"
          },
          {
            "RefID": "CVE-2022-22818",
            "RefURL": "https://nvd.nist.gov/vuln/detail/CVE-2022-22818",
            "Source": "CVE"
          },
          {
            "RefID": "CVE-2022-23833",
            "RefURL": "https://nvd.nist.gov/vuln/detail/CVE-2022-23833",
            "Source": "CVE"
          }
        ],
        "Description": "This update upgrades python3-module-django to version 3.2.12-alt1. \nSecurity Fix(es):\n\n * BDU:2022-00584: Уязвимость фреймворка для веб-приложений Django, связанная с непринятием мер по защите структуры веб-страницы, позволяющая нарушителю провести атаку межсайтового скриптинга\n\n * BDU:2022-00585: Уязвимость фреймворка для веб-приложений Django, связанная с выполнением цикла с недоступным условием выхода, позволяющая нарушителю выполнить отказ в обслуживании\n\n * CVE-2022-22818: The {% debug %} template tag in Django 2.2 before 2.2.27, 3.2 before 3.2.12, and 4.0 before 4.0.2 does not properly encode the current context. This may lead to XSS.\n\n * CVE-2022-23833: An issue was discovered in MultiPartParser in Django 2.2 before 2.2.27, 3.2 before 3.2.12, and 4.0 before 4.0.2. Passing certain inputs to multipart forms could result in an infinite loop when parsing files.",
        "Advisory": {
          "From": "errata.altlinux.org",
          "Severity": "High",
          "Rights": "Copyright 2023 BaseALT Ltd.",
          "Issued": {
            "Date": "2022-02-25"
          },
          "Updated": {
            "Date": "2022-02-25"
          },
          "bdu": [
            {
              "Cvss": "AV:N/AC:L/Au:N/C:P/I:P/A:N",
              "Cvss3": "AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
              "Cwe": "CWE-79",
              "Href": "https://bdu.fstec.ru/vul/2022-00584",
              "Impact": "Low",
              "Public": "20220201",
              "CveID": "BDU:2022-00584"
            },
            {
              "Cvss": "AV:N/AC:H/Au:N/C:N/I:N/A:C",
              "Cvss3": "AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "Cwe": "CWE-835",
              "Href": "https://bdu.fstec.ru/vul/2022-00585",
              "Impact": "Low",
              "Public": "20220201",
              "CveID": "BDU:2022-00585"
            }
          ],
          "Cves": [
            {
              "Cvss": "AV:N/AC:M/Au:N/C:N/I:P/A:N",
              "Cvss3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
              "Cwe": "CWE-79",
              "Href": "https://nvd.nist.gov/vuln/detail/CVE-2022-22818",
              "Impact": "Low",
              "Public": "20220203",
              "CveID": "CVE-2022-22818"
            },
            {
              "Cvss": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
              "Cvss3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "Cwe": "CWE-835",
              "Href": "https://nvd.nist.gov/vuln/detail/CVE-2022-23833",
              "Impact": "High",
              "Public": "20220203",
              "CveID": "CVE-2022-23833"
            }
          ],
          "AffectedCpeList": {
            "Cpe": [
              "cpe:/o:alt:kworkstation:10",
              "cpe:/o:alt:workstation:10",
              "cpe:/o:alt:server:10",
              "cpe:/o:alt:server-v:10",
              "cpe:/o:alt:education:10",
              "cpe:/o:alt:slinux:10",
              "cpe:/o:alt:starterkit:p10",
              "cpe:/o:alt:kworkstation:10.1",
              "cpe:/o:alt:workstation:10.1",
              "cpe:/o:alt:server:10.1",
              "cpe:/o:alt:server-v:10.1",
              "cpe:/o:alt:education:10.1",
              "cpe:/o:alt:slinux:10.1",
              "cpe:/o:alt:starterkit:10.1",
              "cpe:/o:alt:kworkstation:10.2",
              "cpe:/o:alt:workstation:10.2",
              "cpe:/o:alt:server:10.2",
              "cpe:/o:alt:server-v:10.2",
              "cpe:/o:alt:education:10.2",
              "cpe:/o:alt:slinux:10.2",
              "cpe:/o:alt:starterkit:10.2"
            ]
          }
        }
      },
      "Criteria": {
        "Operator": "AND",
        "Criterions": [
          {
            "TestRef": "oval:org.altlinux.errata:tst:2001",
            "Comment": "ALT Linux must be installed"
          }
        ],
        "Criterias": [
          {
            "Operator": "OR",
            "Criterions": [
              {
                "TestRef": "oval:org.altlinux.errata:tst:20221372001",
                "Comment": "python3-module-django is earlier than 0:3.2.12-alt1"
              },
              {
                "TestRef": "oval:org.altlinux.errata:tst:20221372002",
                "Comment": "python3-module-django-dbbackend-mysql is earlier than 0:3.2.12-alt1"
              },
              {
                "TestRef": "oval:org.altlinux.errata:tst:20221372003",
                "Comment": "python3-module-django-dbbackend-oracle is earlier than 0:3.2.12-alt1"
              },
              {
                "TestRef": "oval:org.altlinux.errata:tst:20221372004",
                "Comment": "python3-module-django-dbbackend-postgresql is earlier than 0:3.2.12-alt1"
              },
              {
                "TestRef": "oval:org.altlinux.errata:tst:20221372005",
                "Comment": "python3-module-django-dbbackend-sqlite3 is earlier than 0:3.2.12-alt1"
              },
              {
                "TestRef": "oval:org.altlinux.errata:tst:20221372006",
                "Comment": "python3-module-django-doc is earlier than 0:3.2.12-alt1"
              }
            ]
          }
        ]
      }
    }
  ]
}